orion/sites/admin/src/location/location.js

```diff
@@ -20,29 +24,97 @@ export class LocationPage extends React.Component {
         });
     }
 
+    onCheckRow() {
+        if (document.getElementById("unpublished").checked) {
```

Comment:
I thiinnk the problem could be here where you're searching by Id instead of searching by class.

Turns out document.getElementByX can return an array of elements so you'll need to loop over them.
`document.getElementsByClassName('test')[0]`

However, if `document.getElementsByClassName(...)` doesn't find anything, it will return `undefined`.
So I would do something like this:
```
const elements = document.getElementsByClassName(...) || [];
... then loop
```
What this does is if `document.blahblah` is a valid array, use that value! If it becomes undefined, assign it to `[]` instead. This overall idea is a Javascript thing called `falsy values`.